src/modules/app/containers/I18nProvider.js

```diff
@@ -1,11 +1,14 @@
+import i18n from 'i18next'
 import React from 'react'
 import { connect } from 'react-redux'
 import { I18nextProvider } from 'react-i18next'
+import { reduce, forEach } from 'lodash/collection'
 import WebFont from 'webfontloader'
 import PropTypes from 'prop-types'
+import LanguageDetector from 'i18next-browser-languagedetector'
 
-import { LANGUAGE_CODE_LENGTH } from '../constants'
-import i18n from '../i18n'
+import localesResources from 'locales.json'
+import { MAX_LANGUAGE_CODE_LENGTH, MIN_LANGUAGE_CODE_LENGTH } from '../constants'
```

Comment:
Do we still need them since have no ambigious routes any more?

Reply:
I think so, yes. Otherwise we would redirect to the landing page if you enter '/berlin' or '/some_random_stuff', because we have to decide what the meaning of this single param is (language, city, disclaimer, not_found...)

Reply:
Well... I think we should always redirect from '/' to '/landing/{i18nLanguage}'
and from '/asdf' to '/asdf/{i18nLanguage}' ?
So no need to decide if it's a language because it'll always be a city...

Reply:
I will remove this redirect, as discussed